pkg/liqoctl/install/utils.go

```diff
@@ -0,0 +1,75 @@
+package install
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	helm "github.com/mittwald/go-helm-client"
+	"gopkg.in/yaml.v3"
+	"helm.sh/helm/v3/pkg/action"
+	"k8s.io/client-go/rest"
+
+	helm2 "github.com/liqotech/liqo/pkg/liqoctl/install/helm"
+	"github.com/liqotech/liqo/pkg/liqoctl/install/kubeadm"
+	"github.com/liqotech/liqo/pkg/liqoctl/install/provider"
+	"github.com/liqotech/liqo/pkg/utils"
+)
+
+func getProviderInstance(providerType string) provider.InstallProviderInterface {
+	if providerType == "kubeadm" {
+		return kubeadm.NewKubeadmProviderConstructor()
+	}
+	return nil
+}
+
+func initHelmClient() (*helm.HelmClient, *rest.Config, error) {
+	kubeconfigPath, ok := os.LookupEnv("KUBECONFIG")
+	if !ok {
+		kubeconfigPath = filepath.Join(os.Getenv("HOME"), ".kube", "config")
+	}
+
+	config, err := utils.UserConfig(kubeconfigPath)
+	if err != nil {
+		fmt.Printf("Unable to create client config: %s", err)
+		return nil, nil, err
+	}
```

Comment:
Would separate the rest config retrieval from the helm sub-package, as unrelated